src/classes/groundplaces.ts

```diff
@@ -33,27 +34,88 @@ export class GroundPlacesController {
 
   /**
    * @description Init GroundPlaces file.
-   * @param {string|undefined} groundPlacesFile - The file to manipulate, can only be JSON for now.
+   * @param {GroundPlacesFile|undefined} groundPlacesFile
+   * The file to manipulate, can only be JSON for now.
+   *
    * If you provide empty value, the default file will be the file retrieved from Amazon S3.
+   * @returns {void}
    */
-  public init(groundPlacesFile?: GroundPlaces): void {
+  public init(groundPlacesFile?: GroundPlacesFile): void {
     if (groundPlacesFile) {
       this.storageService.initFile(groundPlacesFile);
     } else {
-      const groundPlacesFileS3: GroundPlaces = this.webService.downloadDistantGroundPlacesMaster();
+      const groundPlacesFileS3: GroundPlacesFile = this.webService.downloadDistantGroundPlacesMaster();
 
       this.storageService.initFile(groundPlacesFileS3);
     }
   }
 
   /**
-   * @description Returns a list of places.
+   * @description Returns a list of ground places.
    * @param {string} query - Can be a name, a Gpuid, a unique name or other name.
-   * @param {AutoCompleteFilters} filters - Filters with different options (StopGroup, StopCluster, Serviced, SegmentProvider).
-   * @returns {AutoComplete[]}
+   * @param {AutoCompleteFilters[]|undefined} filters - Filters with different options (StopGroup, StopCluster, Serviced, SegmentProvider).
+   *
+   * If you do not give filters, the list will not be filtered.
+   * @returns {GroundPlace[]}
    */
   // @ts-ignore
-  public autocomplete(query: string, filters: AutoCompleteFilters[]): AutoComplete[] {}
+  public autocomplete(query: string, filters?: AutoCompleteFilters[]): GroundPlace[] {
```

Comment:
if it's an array of filter no S at AutoCompleteFilterS (and i think it should be AutocompleteFilter (no C capitalized C)